gcloud-java-logging/src/main/java/com/google/cloud/logging/spi/v2/LoggingServiceV2Api.java

```diff
@@ -381,15 +319,18 @@ private void deleteLog(DeleteLogRequest request) {
    * Example: `{ "size": "large", "color":"red" }`
    * @param entries Required. The log entries to write. The log entries must have values for
    * all required fields.
+   *
+   * To improve throughput and to avoid exceeding the quota limit for calls
+   * to `entries.write`, use this field to write multiple log entries at once
+   * rather than  // calling this method for each log entry.
```

Comment:
It appears that way in the original proto:

https://github.com/googleapis/googleapis/blob/master/google/logging/v2/logging.proto#L89

This might be due to transformations that happen during the open sourcing process. 

Filed issue https://github.com/googleapis/googleapis/issues/85 .